tests/optimize/test_qng.py

```diff
@@ -277,13 +328,14 @@ def gradient(params):
 
             # check metric tensor
             res = opt.metric_tensor
-            exp = np.diag([0.25, (np.cos(x) ** 2) / 4])
-            assert np.allclose(res, exp, atol=0.00001, rtol=0)
+            exp = np.array([[[0.25]], [[(np.cos(2 * theta[0]) + 1) / 8]]])
+            assert np.allclose(res, exp)
```

Comment:
This test, which was deactivated via `skip`, assumed that the optimizer will use the metric tensor for all arguments taken together. However ,we do not do that, so `exp` needs to be a 2-tuple of smaller `(1,1)` arrays.